### PR TITLE
This is not a vulnerability and was assigned against CNA guidelines

### PR DESCRIPTION
**Pull request by iamamoose**
For commentary see https://github.com/openssl/openssl/issues/16873

### CODE REVIEW

File: 2021/43xxx/CVE-2021-43085.json [hunk 1]
```diff
@@ -2,30 +2,7 @@
     "CVE_data_meta": {
```
Comment (cve-team) [inline 2021/43xxx/CVE-2021-43085.json:2]: CNA(s) of user (apache|openssl) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/43xxx/CVE-2021-43085.json (hunk 1 above)
Comment (iamamoose) [reply 2021/43xxx/CVE-2021-43085.json:2]: Please have owning CNA reassign this CVE to OpenSSL CNA due to breach of CNA guidelines.

**Comment by anthnysingleton** [comment]
Hello @iamamoose,

We have resolved the issue on our end [here](https://github.com/CVEProject/cvelist/blob/acbb5be766e78210bee16c711ae74981f213cd69/2021/43xxx/CVE-2021-43085.json) and the CNA Coordination Team will be reaching out to you shortly. 

Thanks,
CVE Content Team

**Pull request closed without merging**